### PR TITLE
2061650 Update "MTV upgrade" section in documentation

### PR DESCRIPTION
**Pull request by stoobie**
This fixes https://bugzilla.redhat.com/show_bug.cgi?id=2061650

Target MTV 2.2.

Preview:
https://deploy-preview-286--forklift-documentation.netlify.app/documentation/doc-migration_toolkit_for_virtualization/master/#upgrading-mtv-ui_forklift

### CODE REVIEW

**Comment by stoobie** [comment]
@mguetta1 Please review.

File: documentation/modules/upgrading-mtv-ui.adoc [hunk 1]
```diff
@@ -24,7 +24,7 @@ ifeval::["{build}" == "downstream"]
 See link:https://access.redhat.com/documentation/en-us/openshift_container_platform/{ocp-version}/html-single/operators/index#olm-changing-update-channel_olm-upgrading-operators[Changing update channel] in the {ocp} documentation.
 endif::[]
 
-. Restart the `CatalogSource` pod:
+. On the *Subscriptions* tab of the *Operator details* page in the {ocp-short} web console, confirm that *Upgrade status* changes from *Up to date* to *Upgrade available*. If it does not, restart the `CatalogSource` pod:
```
Comment (mguetta1) [inline documentation/modules/upgrading-mtv-ui.adoc:27]: I think it is better to start with "Confirm that *Upgrade status* changes...."
As the used is already in the right page when execute step 1

And for the same reason 2.a should be removed

**Review by mguetta1 — CHANGES_REQUESTED** [review]

File: documentation/modules/upgrading-mtv-ui.adoc (hunk 1 above)
Comment (stoobie) [reply documentation/modules/upgrading-mtv-ui.adoc]: @mguetta1 If you're already on the *Subscriptions* tab, then that impacts other steps in the procedure as well. Please see my changes.

**Review by mguetta1 — CHANGES_REQUESTED** [review]

File: documentation/modules/upgrading-mtv-ui.adoc [hunk 2]
```diff
@@ -15,6 +15,8 @@ You must upgrade to the next release without skipping a release, for example, fr
 
 .Procedure
 
+. Click the *Subscriptions* tab of the *Operator details* page in the {ocp-short} web console.
```
[the remark this answers is not in the document]
Comment (mguetta1) [reply documentation/modules/upgrading-mtv-ui.adoc:18]: Selection_367

[the remark this answers is not in the document]
Comment (stoobie) [reply documentation/modules/upgrading-mtv-ui.adoc:18]: @mguetta1 How do you change the Update channel? It looks like you can't edit it.

[the remark this answers is not in the document]
Comment (mguetta1) [reply documentation/modules/upgrading-mtv-ui.adoc]: @stoobie It is an old pic, I don't know why the Update channel is disabled.
You can update the channel by clicking on the value in the field. I sent the picture to show where you can change it

File: documentation/modules/upgrading-mtv-ui.adoc [hunk 3]
```diff
@@ -15,6 +15,8 @@ You must upgrade to the next release without skipping a release, for example, fr
 
 .Procedure
 
+. In the {ocp-short} web console, click *Operators*->*Installed Operators*->*{operator-name}*->*Subscription*.
```
Comment (mguetta1) [inline documentation/modules/upgrading-mtv-ui.adoc:18]: @stoobie Hi,
1. Regaring {operator-name}  - shouldn't it be {operator-name-ui} to have the operator name as it is in UI?
2. only the word **Operators** is bold
Selection_371

**Review by mguetta1 — CHANGES_REQUESTED** [review]

File: documentation/modules/upgrading-mtv-ui.adoc (hunk 3 above)
Comment (stoobie) [reply documentation/modules/upgrading-mtv-ui.adoc]: You are correct! Fixed.

Comment (mguetta1) [reply documentation/modules/upgrading-mtv-ui.adoc]: Could you please fix also point number-2 above: only the word 'Operators' is bold?

Comment (stoobie) [reply documentation/modules/upgrading-mtv-ui.adoc]: Yes, sorry I missed that. I didn't realize I needed to add some spaces. Fixed now.

**Review by mguetta1 — APPROVED** [review]

**Comment by mguetta1** [comment]
Looks good to me

**Comment by sbeskin-redhat** [comment]
Looks OK to me

**Pull request merged**